src/explore-education-statistics-common/src/modules/charts/util/getUnit.ts

```diff
@@ -0,0 +1,23 @@
+import { DataSetCategoryConfig } from '@common/modules/charts/util/getDataSetCategoryConfigs';
+
+const getUnit = (
+  configUnit: string | undefined,
```

Comment:
Instead of writing `configUnit: string | undefined` in Typescript, you can write `configUnit?: string` - the question mark shows it's possibly undefined.  We predominantly use this syntax for possibly-undefined properties elsewhere in the codebase I believe.

Reply:
I've made this change. I had to reorder the params to do so.

Reply:
Oh sorry then @benoutram , that was probably a bad steer from me then.  I'll set it back - I was mixing up method parameters with object properties.